src/web/dataadapters/GreyNoiseAdapterDocumentation.jsx

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+/* eslint-disable react/no-unescaped-entities, no-template-curly-in-string */
+import React from 'react';
+
+class GreyNoiseAdapterDocumentation extends React.Component {
+  render() {
+    return () => {
+    };
```

Comment:
We should have something here.  Maybe a quick adaptation of some text from GreyNoise's own documentation.